### PR TITLE
Add first mkdocs setup, add contributing guidelines

### PR DESCRIPTION
**Pull request by alexsavio**
Still need to test if readthedocs support the theme

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+from lambda_handlers.response import cors
+
+@http_handler(
+    cors=cors(origin='localhost', credentials=False),
+)
+def handler(event, context):
+    return event['body']
+```
+
+Using jsonschema to validate a the input of a User model.
```
Comment (sleistner) [inline docs/examples.md]: typo

File: docs/examples.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+@http_handler(
+    validator=validators.jsonschema(body=user_schema),
+)
+def handler(event, context):
+    user = event['body']
+    return user
+```
+
+Using Marshmallow to validate a User model in the input and in
```
Comment (sleistner) [inline docs/examples.md]: second `in` redundant

File: docs/examples.md (hunk 1 above)
Comment (alexsavio) [reply docs/examples.md]: thanks. I think it is fixed.

File: docs/examples.md (hunk 2 above)
Comment (alexsavio) [reply docs/examples.md]: thanks. I think it is fixed.

**Comment by alexsavio** [comment]
btw, what should we do with circleci?

**Pull request merged**